src/pybind/mgr/nfs/cluster.py

```diff
@@ -256,3 +252,7 @@ def reset_nfs_cluster_config(self, cluster_id: str) -> Tuple[int, str, str]:
                 "(Manual Restart of NFS PODS required)", ""
         except Exception as e:
             return exception_handler(e, f"Resetting NFS-Ganesha Config failed for {cluster_id}")
+
+    def _rados(self, cluster_id: str) -> NFSRados:
+        """Return a new NFSRados object for the given cluster id."""
+        return NFSRados(self.mgr.rados, cluster_id)
```

Comment:
I think the code probably deserves a bit of a general, but not huge, reorganization. Since you already approved I won't make any reorg'ing changing in this PR, but we can come back to it in another future cleanup pr for this too.